bugzilla_mcp/utils/bugzilla.py

```
@@ -58,7 +60,818 @@ async def add_comment(
+        downloaded = []
+        for att in attachments:
+            att_id = att.get("id")
+            file_name = att.get("file_name")
+            b64_data = att.get("data")
+
+            if not b64_data:
+                continue
+
+            try:
+                file_content = base64.b64decode(b64_data)
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)